ni_measurementlink_service/measurement/service.py

```diff
@@ -375,14 +375,16 @@ def host_service(self) -> MeasurementService:
         """
         if self.measure_function is None:
             raise Exception("Error, must register measurement method.")
-        self.grpc_service.start(
+        is_service_started = self.grpc_service.start(
```

Comment:
`start` function returns `str` or `None`. 

`is_service_started` doesn't seem to fit in here.. Can we rename to `port_number` or `port` itself